packages/nextjs-kit/src/components/header.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Link from 'next/link';
+import { Headers } from '../index';
+
+interface HeaderProps {
+  headers: Headers[];
+}
+
+/**
+ * This is a Header component.
+ *
+ * @param {Props} headers holds the title and href of each route to be displayed
+ * @returns {React.JSX.Element}
+ */
+
+const Header: React.FC<HeaderProps> = ({ headers }: HeaderProps) => {
```

Comment:
I am starting to think we should be explicit on the return of the function signature too:
```suggestion
const Header: React.FC<HeaderProps> = ({ headers }: HeaderProps): JSX.Element => {
```